### PR TITLE
[PIL-315] - Added logic to add a new record called in the test controller PIL-315

### PR DESCRIPTION
**Pull request by olakunlekuye2**

### CODE REVIEW

File: conf/prod.routes [hunk 1]
```diff
@@ -3,3 +3,6 @@
 ->         /                          health.Routes
 
 GET        /admin/metrics             com.kenshoo.play.metrics.MetricsController.metrics
+
+# Include Test Routes
+->    /test                       test.Routes
```
Comment (AKKUM) [inline conf/prod.routes:8]: We dont need to add here.

**Comment by AKKUM** [comment]
Could not see the configuration change where TTL was only considering for days. That may need to change for seconds.

File: conf/prod.routes (hunk 1 above)
Comment (olakunlekuye2) [reply conf/prod.routes]: Done

**Review by AKKUM — APPROVED** [review]

**Review by AKKUM — DISMISSED** [review]

File: app/uk/gov/hmrc/pillar2/config/AppConfig.scala [hunk 2]
```diff
@@ -26,7 +26,6 @@ class AppConfig @Inject() (val config: Configuration, servicesConfig: ServicesCo
   val appName: String = config.get[String]("appName")
 
   val defaultDataExpireInSeconds = config.get[Int]("defaultDataExpireInSeconds")
```
Comment (jassalrichy) [inline app/uk/gov/hmrc/pillar2/config/AppConfig.scala:28]: defaultDataExpireInSeconds is set to 60 in application.conf ?

**Review by jassalrichy — DISMISSED** [review]

**Review by jassalrichy — DISMISSED** [review]

**Review by javed-iqbal1 — APPROVED** [review]

**Pull request closed without merging**